src/rest/CommunityEndpoints.js

```diff
@@ -28,87 +204,286 @@ export async function joinCommunity(communityAddress, memberAddress, secret = un
     )
 }
 
-export async function memberStats(communityAddress, memberAddress) {
-    const url = `${this.options.restUrl}/communities/${communityAddress}/members/${memberAddress}`
-    return fetch(url).then((res) => res.json())
-}
+// //////////////////////////////////////////////////////////////////
+//          member: JOIN & QUERY COMMUNITY
+// //////////////////////////////////////////////////////////////////
 
-export async function withdraw(communityAddress, memberAddress, wallet, confirmations = 1) {
-    const stats = await this.memberStats(communityAddress, memberAddress)
-    if (!stats.withdrawableBlockNumber) {
-        throw new Error('No earnings to withdraw.')
+/**
+ * Send a joinRequest, or get into community instantly with a community secret
+ * @param {EthereumAddress} communityAddress to join
+ * @param {String} secret (optional) if given, and correct, join the community immediately
+ */
+export async function joinCommunity(communityAddress, secret) {
```

Comment:
I think we should add `partCommunity` or similar so that members themselves can leave the community.

Reply:
good idea actually! Only I'm not sure if it's even implemented in EE currently (probably not), so... it would have to be done by admin anyway, for which there is `kick`.